### PR TITLE
{vis}[GCCcore/10.2.0] GLFW v3.3.3

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/bfc24ff3aa467362c0febb3d6ec64798 for a full test report.

File: easybuild/easyconfigs/g/GLFW/GLFW-3.3.3-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+# build both static and shared libraries
+configopts = [
+    "-DBUILD_SHARED_LIBS=OFF",
+    "-DBUILD_SHARED_LIBS=ON",
+]
+
+sanity_check_paths = {
+    'files': ['include/GLFW/glfw3.h', 'include/GLFW/glfw3native.h',
+              'lib/libglfw3.a', 'lib/libglfw.%s' % SHLIB_EXT , 'lib/pkgconfig/glfw3.pc'],
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/g/GLFW/GLFW-3.3.3-GCCcore-10.2.0.eb:34]: the style check error is `E203 whitespace before ','` (after `SHLIB_EXT`)

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/b4c67025d585c5f870e0e8d9f923d60e for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/a4e608ce0845526c62b105bac38c80f9 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.2
See https://gist.github.com/2207e302d11dd94acce666d521f7dd3a for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @Micket!

**Pull request merged**